<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hello</class>
 <widget class="QWidget" name="Hello">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Привіт мамо !!!!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="bnLink">
     <property name="text">
      <string>Зв’язок</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="Ai_0">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_1"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_2"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_3"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_4"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_5"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_6"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_7"/>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="Ai_8">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_9"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_10"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_11"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_12"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_13"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_14"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Ai_15"/>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>121</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="cbOut_1">
              <property name="text">
               <string>Out 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbOut_2">
              <property name="text">
               <string>Out 2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbOut_3">
              <property name="text">
               <string>Out 3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbOut_4">
              <property name="text">
               <string>Out 4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbOut_5">
              <property name="text">
               <string>Out 5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbOut_6">
              <property name="text">
               <string>Out 6</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbOut_7">
              <property name="text">
               <string>Out 7</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Y_nup">
        <property name="styleSheet">
         <string notr="true">QCheckBox{
min-width: 14px;
min-height: 14px;
max-width: 14px;
max-height: 14px;
}
QCheckBox:checked
{
	
	image: url(:/big/ar_up_on_big.png);
}

QCheckBox:unchecked
{
	image: url(:/big/ar_up_off_big.png);
	
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 14px;
	width: 14px;
}
</string>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Y_ndw">
        <property name="styleSheet">
         <string notr="true">QCheckBox{
min-width: 14px;
min-height: 14px;
max-width: 14px;
max-height: 14px;
}
QCheckBox:unchecked
{
	
	image: url(:/big/ar_down_off_big.png);
}
QCheckBox:checked
{
	
	image: url(:/big/ar_down_on_big.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 14px;
	width: 14px;
}
</string>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="trend">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineDate">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Температура</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineT_1">
        <property name="font">
         <font>
          <pointsize>80</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>23,5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hello.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Hello</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
